pkg/api/norman/customization/project/project_actions.go

```diff
@@ -52,6 +55,7 @@ type Handler struct {
 	ProjectLister     v3.ProjectLister
 	ClusterManager    *clustermanager.Manager
 	ClusterLister     v3.ClusterLister
+	ClusterListerEx   ctrlv1.ClusterController
```

Comment:
This member needs a better name

Reply:
Open to suggestions/standards from other places in the codebase

Reply:
Well, you aren't really using a `Lister` here, so maybe something more like `ProvisioningClusterController` or `ProvisioningClusterClient`? Also, have  you tried to see if you can use a cache for this? would be more efficient than a controller/client.

Reply:
Renamed and used a cache.